lib/wahwah/helper.rb

```diff
@@ -54,6 +54,8 @@ def self.file_format_from_extension(file_path)
     def self.file_format_from_signature(io)
       io.rewind
       signature = io.read(16)
+      # Rewind after reading the signature to not mess with the file position
+      io.rewind
```

Comment:
@aidewoode Added `rewind` to the helper (I'll also be adding it to the `Tag` class as discussed)